epoch/api/channels_api_usage.md

```diff
@@ -173,18 +173,26 @@ The initiator receives the following message
  }
 ```
 
+#### Signed channel_create_tx
+Both participans receive the co-signed `channel_create_tx`:
+```
+{'action': 'on_chain_tx',
+ 'payload': {'tx': 'tx$9mmqY8QRoYS8QR5nyRkWwTyhwecE8NzHotRK1yMjvfvyPfsaBNwWFDrmzG3M5rHKARm39AqeJpgN4Aj7V57yP4KusptAvMA2r3y593vr51ubXN8zo8t8qg7Arb32wWhmYDvRXVGKhfYoqu91BpkammeY69xCjGuQrXKDk3t9aPCcrvM3PbERAMvkmvD9dsb2H5iujGin8qZTy42xsUYS6QcTiaomxo8CmieGSvZkAai1KCZ84MZzsAjsR1FP54Su7rfBPXZTd5u2AmyF7LwZoHQsJEspLnzXxun2MtKTkhR3du8PY'
+            }
+}
+```
+Using its hash participants can track its progress on the chain - entering the mempool, block inclusion and a number of confirmations.
```

Comment:
"Using its hash, participants can track its progress on the chain: entering the mempool, block inclusion ..." (comma after "hash", replace "-" with ":")